Makefile

```diff
@@ -398,6 +396,7 @@ add-on: node_modules \
 	$(RSYNC) $@/chrome/test $(BUILT)/$@/chrome
 	mkdir -p $(BUILT)/$@/chrome/content/shared
 	$(RSYNC) --exclude='*.js' shared $(BUILT)/$@/chrome/content
+	$(RSYNC) $(BUILT)/shared $(BUILT)/$@/chrome/content
```

Comment:
The other places this added line is just after `mkdir` so let's be consistent here.